README.md

```diff
@@ -22,49 +34,152 @@ You can do it manually following the instructions from [rust-lang.org](https://w
 
-Please use clang-format 11.0.0 with provided [.clang-format](.clang-format) file to autoformat the code.  
+### Fuhon-node and Lotus-miner
+1. Establishing of a Fuhon node is described in **Fuhon miner establishing**
+2. Lotus miner can be established by the following way:
+    - to begin with you need you own bls wallet(described in **Working with Lotus-CLI**)
+    - init your miner with command: `lotus-miner --repo <your node running directory> --miner-repo <minner future working directory> init --nosync --owner <your bls wallet public address> --sector-size <Size of sectors that will be sealed>`
+    - to know more about lotus software establishing, read [filecoin-docs](https://docs.filecoin.io/get-started/lotus/installation/#minimal-requirements "lotus establishing guide")
 
-## Maintenance
+## Docker configuration
+    Will be soon!
 
-Maintainers: @kamilsa, @zuiris, @harrm, @masterjedy, @Mingela, @Warchant
+## Your new Filecoin network start
 
-Tickets: Can be opened in Github Issues.
+To run a new filecoin network, you should run at least two nodes, that will have communication with each other, and a miner (miners are representation of the network power, so more miners connected and sealed, more power your network will have).
+
+1. To run the first node you will need to generate a new genesis file which will be an initial instruction for all new conected nodes.
+   This can be done with lotus-seed tools:
+    - Clone  [lotus repository](https://github.com/filecoin-project/lotus "lotus project")
+
+    - build lotus-seed with `$make lotus-seed` or `make debug` - for debug version initiation,
+      or `make 2k` for version with 2KiB sector size  or `make interopnet` for interopnet version.
+
+    - Generate genesis and preseal. Ensure lotus-seed was build in previous step and run script node `$genesis.js DIR 1` to generate genesis. `DIR` - directory where genesis and presealed sectors will be generated, 1 - number of sectors for miner. In case your lotus executables lay in custom directory or sript can not find them, you can link lotus target with `LOTUS=/<path to your lotus executable>/lotus node genesis.js <directory where results wil be> <sealing sectors upper bound>`
+
+2. Run your node-1 with new genesis file as it was done in chapter **Fuhon node establishing**
+
+3. Run your node-2 as the 1st one, but be sure to open api port on it with `--api <api port>`
+
+4. Connect node-2 to node-1 by commands:
+   `addr=$(node --repo "REPO1" net listen | grep 127.0.0.1)`
+   `$node --repo "REPO2" net connect $addr`t 
+
+5. After generating new genesis file, you will also receive genesis.json with declared miners list, these are addresses of the initial miners.
+
+6. To start miner that is declared in genesisfile (step 5) you can use chapter **Fuhon miner establishing** but instead of owner address pass declared in the genesis one.
```

Comment:
`lotus-miner` flags are different for genesis miner

Reply:
also don't understand

Reply:
genesis miner requires `--genesis-miner --actor ACTOR --pre-sealed-sectors PRESEAL_DIR --pre-sealed-metadata PRESEAL_JSON` flags

Reply:
done